website/project_styleguide/templates/patterns/base.html

```diff
@@ -35,6 +35,14 @@
         <script src="{% static 'js/main.js' %}"></script>
 
         {% block extra_js %}{% endblock %}
+        <script>
+            function selectSearchBox() {
+                setTimeout(() => {
+                    document.getElementById("navbarSearch").focus();
```

Comment:
I am not certain of the accessibility implications of enforcing focus on an element directly after page load.